frontend/components/forms/fields/InputFieldWithIcon/_styles.scss

```diff
@@ -20,8 +20,8 @@
   }
 
   &__input {
-    border: 0;
-    border-bottom: 2px solid $brand-ultralight;
+    border: 2px;
```

Comment:
border: 2px what? Missing all the important bits for a border. Color, shape, etc.

Reply:
This actually sets the border for the text entry fields. I'm not entirely sure it works, but it do.